src/main/java/net/library/service/BookService.java

```diff
@@ -52,16 +54,14 @@ public class BookService {
      * If no available book item is found (i.e., no records match the 'AVAILABLE' status),
      * a 'NOT FOUND' error is thrown.
      */
-
     public BookItemDto addBookItem(final BookItemRequest bookItem) {
```

Comment:
According to Java conventions, constant fields (static final) are usually declared before instance fields. move DUE_DATE above the repositories
use please this rule everywhere